server.js

```diff
@@ -0,0 +1,30 @@
+
+const PORT = 3000;
+
+const express = require('express');
+const compression = require('compression');
+const path = require('path');
+
+const app = express();
+const buildDir = path.join(__dirname, '/build');
+
+const singlePageApp = () => {
+  app.get('*', (req, res) => {
+    res.sendFile(`${buildDir}/index.html`);
+  });
+};
+const start = () => {
+  app.listen(PORT, error => {
+    if (error) {
+      console.error(error);
+    } else {
+      console.info(`Express is listening on http://localhost:${PORT}\n`);
+    }
+  });
+};
+
+app.use(compression());
+app.use(express.static('build'));
+
+singlePageApp();
```

Comment:
I'm curious what benefit you see in creating the `singlePageApp` and `start` functions given that each is only called once and each contains a single function call. I'd just move that code to here and remove the function definitions. Otherwise 👍